src/activities/menu/Menu.qml

```diff
@@ -47,6 +47,9 @@ ActivityBase {
     id: activity
     focus: true
     activityInfo: ActivityInfoTree.rootMenu
+    // set to true when returned to home from a dialog.
+    property bool returnFromDialog: false
+
     onBack: {
```

Comment:
I think it should also be added there (I don't have mouse with back button but I think it will trigger this signal)

Reply:
This signal is triggered only on back pressed from phones or other touch devices. Currently, back button from the mouse is not enabled.